src/content/developers/docs/storage/index.md

```diff
@@ -1,18 +1,15 @@
 ---
 title: Decentralized Storage
-description: Overview of what decentralized storage is and available tools to integrate into a dapp.
+description: Overview of what decentralized storage is and the available tools to integrate it into a dapp.
 lang: en
 sidebar: true
-incomplete: true
 ---
 
-# Introduction
+Unlike a centralized server operated by a single company or organization, decentralized storage systems consist of a peer-to-peer network of user-operators who hold a portion of the overall data, creating a resilient file storage sharing system. These can be in a blockchain-based application or any peer-to-peer-based network.
 
-As opposed to a centrally located server operated by a single company or organization, decentralized storage systems consist of a peer-to-peer network of user-operators who hold a portion of the overall data, creating a resilient system of file storage and sharing. These can be in a blockchain-based application, or any peer-to-peer based network.
+Ethereum itself can be used as a decentralized storage system, and it is when it comes to code storage in all the smart contracts. However, when it comes to large amounts of data, that isn't what Ethereum was designed for. At the time of writing, the Ethereum chain is around 350GB, and every node on the network needs to be able to store all 350GB of data. If the chain were to expand to large amounts of data (say 5TBs) it wouldn't be feasible for all nodes to continue to run. Also, this can get incredibly expensive to deploy new data due to the size.
```

Comment:
> At the time of writing, the Ethereum chain is around 350GB, and every node on the network needs to be able to store all 350GB of data.

Where did we get this figure? I feel like we should probably use Geth full nodes as the benchmark - https://etherscan.io/chartsync/chaindefault

Reply:
Yeah this is an outdated figure. It also keeps expanding, and is dependent on client implementation... maybe we put in a range here of ~500GB to ~1TB at time of writing, perhaps linking to this chart. 

```suggestion
Ethereum itself can be used as a decentralized storage system, and it is when it comes to code storage in all the smart contracts. However, when it comes to large amounts of data, that isn't what Ethereum was designed for. The chain is steadily growing, but at the time of writing, the Ethereum chain is around 500GB - 1TB ([depending on the client](https://etherscan.io/chartsync/chaindefault)), and every node on the network needs to be able to store all of the data. If the chain were to expand to large amounts of data (say 5TBs) it wouldn't be feasible for all nodes to continue to run. Also, the cost of deploying this much data to Mainnet would be prohibitively expensive do to [gas](/developers/docs/gas) fees.
```